### DISCUSSION TITLE
Maintaining the sequence of folders for different group of APIs and Sub APIs

### DISCUSSION THREAD

**Discussion by sumitriotinto** [apideck-libraries/portman#363] [Q&A]
Hey Team ,

Currently I am facing one challenge that folder structure for my APIs gets removed once I inject the test operations with portman and import the generated collection file to the postman.

Is there any possible way to maintain the folder structure using portman

Before :
 
![Before](https://user-images.githubusercontent.com/88074356/178288195-e2f4992f-5734-4f7e-834d-fc0259d817be.PNG)

After :

![After](https://user-images.githubusercontent.com/88074356/178288079-56a71d73-b2fc-4458-a4ae-a0063820d985.png)

**Comment by thim81** [comment]
hi @sumitriotinto,

This is a really strange, since Portman typically would not change the folder names. 
It might be linked to the `{id}` folder name.

Would it be possible to share your OpenAPI or Postman source, that you are using to inject tests in and the Portman config?
That way we can try to reproduce it.
